pubsub/gossipsub/README.md

```diff
@@ -191,33 +197,33 @@ towards them.
 ### Control messages
 
 The protocol defines four control messages:
-- `GRAFT`: graft a mesh link; notifies the peer that it has been added to the local mesh view.
-- `PRUNE`: prune a mesh link; notifies the peer that it has been removed from the local mesh view.
-- `IHAVE`: gossip; notifies the peer that the following messages were recently seen and are available on request.
-- `IWANT`: requests the transmission of messages announced in an `IHAVE` message.
+- `GRAFT`: graft a mesh link; this notifies the peer that it has been added to the local mesh view.
+- `PRUNE`: prune a mesh link; this notifies the peer that it has been removed from the local mesh view.
+- `IHAVE`: gossip; this notifies the peer that the following messages were recently seen and are available on request.
+- `IWANT`: request out of band transmission of messages announced in an `IHAVE` message. (Again, in-band transmission isn't specified; [peer routing](https://github.com/libp2p/specs/blob/master/) can be done by any protocol implementation such as kad or mDNS.)
 
 ### Router state
 
 The router maintains the following state:
 - `peers`: a set of all known peers; `peers.gossipsub` denotes the gossipsub peers
    while `peers.floodsub` denotes the floodsub peers.
-- `mesh`: the overlay meshes as a map of topic to lists of peers.
+- `mesh`: the overlay meshes as a map of topics to lists of peers.
 - `fanout`: the mesh peers to which we are publishing to without topic membership,
-   as a map of topic to list of peers.
-- `seen`: this is the timed message id cache, which tracks seen messages.
+   as a map of topics to lists of peers.
+- `seen`: this is the timed message `id` cache, which tracks seen messages.
```

Comment:
why the code-ization of id? You are doing it in a few places, and I don't quite like it aesthetically.

Reply:
In words, ID is preferable, but in code, `id` is better because variables are lower case. So pick which one you prefer, but preferably not id.

Reply:
please avoid the code-ification.